test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb

```diff
@@ -283,8 +283,8 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
         end
         should "make outcome calculations" do
           assert_state_variable "total_hours", 45
-          assert_state_variable "minimum_hourly_rate", 6.08
-          assert_state_variable "total_hourly_rate", 6.07
+          assert_state_variable "minimum_hourly_rate", 6.5
+          assert_state_variable "total_hourly_rate", "6.12"
```

Comment:
I can see that `total_hourly_rate` calculation is [covered](https://github.com/alphagov/smart-answers/blob/2620dc0ab2f4d477fc0f9d01f1e02bc63c57ceed/test/unit/calculators/minimum_wage_calculator_test.rb#L85) in unit tests. Then the only thing that it's testing here is that those values are exposed as state variables, which is indirectly covered by regression tests I believe. Therefore I think it should be OK get rid of this test.